### ISSUE TITLE
[1.10.2] Bad Looting drop calculation.

### ISSUE THREAD

**Issue by AfterRebelion** [Ipsis/Woot#124]
Woot Version: 1.2.1
Forge Version: 12.18.3.2185
Only Woot Installed.

After testing with a Woot Blaze spawner, I come to the conclusion that Woot doesn't get looting enchantments loot very well.
![image](https://cloud.githubusercontent.com/assets/559819/21063259/efca9780-be54-11e6-9f75-e96b0128820b.png)
As you can see, Woot gets a little over 90% drop rate, and then I tested with a vanilla diamond sword with a Looting 3 enchantment:
![image](https://cloud.githubusercontent.com/assets/559819/21063331/510b3f7c-be55-11e6-9ffa-9010cb7e5094.png)
This was with 2 round of 10 blaze each.
This makes it into 200% drop rate, instead of the 90% drop rate that Woot is getting.

**Comment by Ipsis** [comment]
That is a bit strange, as when Woot is learning drops, it kills with a diamond sword and a looting I/II/III enchant.

I'll have to investigate this and see if I can identify the issue.

**Comment by Ipsis** [comment]
I had a bit of a dig and you will never see >100% drops from Woot's information.

The number you see in that debug message is the chance of a "blaze rod drop".
The detail version of the command shows you the range of stack sizes and their weight for that "blaze rod drop".
Each time that the blaze rod chance is hit, then there is a second check for which stack size. Woot learns not only the drops, but how many of each item was dropped.

eg.
/woot loot Woot:none:Blaze detail
Woot:none:Blaze/Looting III: 210 [ 194 x Blaze Rod @ 92.38% (s1:w52)(s3:w70)(s2:w65)(s4:w7)

So you have a 92.39% change of getting a Blaze Rod drop with Looting III, in the sampling that I did. That drop can then be a stack size of 1,2,3 or 4, where the weighting will decide on which one is chosen.

If you get you 92 drops from 100 kills and each of those drops are lucky enough to get a stack size of 4, then you get 368 blaze rods from 100 blaze.

**Comment by AfterRebelion** [comment]
The problem is that with this setting:
![image](https://cloud.githubusercontent.com/assets/559819/21074933/17f17baa-bf06-11e6-8707-86ff465a6420.png)
I should get 16~ blaze rods every 4 seconds (80 ticks), but I only get 8 blaze rods at max.

**Comment by Ipsis** [comment]
Can you dump the fulll /woot loot Woot:none:Blaze detail so i can see the weights and I'll load up the configuration in my dev area.

**Comment by AfterRebelion** [comment]
I can give you de loot.json config:
https://gist.github.com/AfterRebelion/28c65313c5b994d1362075dc96d7ddfe

**Comment by Ipsis** [comment]
Found the problem. It is not to do with the learning, it is when the items are being placed in the inventory.

Instead of placing all the drops into the inventory it is only placing a stack of size one for each entry.
So in my debug, I had 8 stacks of blaze rods of varying sizes.
However for each of those stacks it put one blaze rod in the chest and therefore I got 8 blaze rods.

So I need to fix the inventory code.
